enhancements/multi-arch/multiarch-manager-operator.md

```diff
@@ -333,14 +335,12 @@ type PodPlacementConfigSpec struct {
 	// +kubebuilder:default=Normal
 	LogVerbosity LogVerbosityLevel `json:"logVerbosity,omitempty"`
 
-	// ExcludedNamespaces is the list of namespaces the pod placement controller should ignore.
-	// When omitted, this means no opinion and the operator is left to choose a reasonable default.
-	// The current default behaviour is to ignore only reserved namespaces.
-	// Reserved namespaces are those matching the following globs; openshift-*,kube-*,hypershift-*.
-	// The list of reserved namespaces will always be excluded, independent of the value of this field.
-	// The list of reserved namespaces is subject to change over time.
```

Comment:
Do we still want this behaviour with the namespace selector? Or are we dropping the restriction of not applying this to openshift/kube/hypershift namespaces?

Reply:
openshift/kube/hypershift namespaces are excluded implicitly based on the namespace name. Unfortunately there is no specific label; we can use to exclude these (I think). this is already stated I believe.

Reply:
The information about excluding these was dropped from the godoc in the latest update, so we should make sure it makes its way back in

Reply:
Fixed.